dist/module/locale.js

```diff
@@ -427,7 +427,7 @@ export var COUNTRY_LANGS = {
   TR: [LANG.TR, LANG.EN],
   TT: [LANG.EN, LANG.FR, LANG.ES, LANG.ZH],
   TV: [LANG.EN, LANG.FR, LANG.ES, LANG.ZH],
-  TW: [LANG.ZH, LANG.EN],
+  TW: [LANG.ZH, LANG.EN, LANG.ZH_HANT],
```

Comment:
This looks OK, although the first value in the list will be the default, which will be Simplified Chinese if no other context attribute is found.  Do you want `LANG.ZH_HANT` to be first, before `LANG.ZH` ?

Reply:
Sure! Let me make the change!